growth-ring/src/wal.rs

```diff
@@ -918,7 +927,11 @@ impl WalLoader {
     }
 
     fn verify_checksum_(data: &[u8], checksum: u32, p: &RecoverPolicy) -> Result<bool, WalError> {
-        if checksum == CRC32.checksum(data) {
+        let mut hasher = Hasher::new();
```

Comment:
nit: rather than do this 3 times, an inline function would perform the same and be less code.